### DISCUSSION TITLE
Velocidad arduino vs esp8266

### DISCUSSION THREAD

**Discussion by jchunly** [revoxhere/duino-coin#1224] [Q&A]
Hola. Tengo una esp6288 la cual está minando a 10kh/s. Estoy pensando en comprar otra esp8266/esp32, o un arduino.
He visto por videos que arduino solo tiene una velocidad de 100h/s. Siendo esto así, es más rentable adquirir esp32.
Opiniones?
Saludos.

**Comment by LDarki** [comment]
Hola, perdona por responder algo tarde.

En Duino-Coin mientras menos hashrate más ganas, ya que Kolka está diseñado para ser minado en dispositivos pequeños como un Arduino o un ATTiny.

PD: La próxima vez que necesites ayuda o tengas alguna duda hazla en el [Discord](https://discord.gg/HfDEKzdhQ5) ya que ahí hay bastante gente que habla español.

Saludos Atte : LDarki
